com.biglybt.BiglyBT.metainfo.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Patrizio Bruno <desertconsulting@gmail.com> -->
+<component type="desktop">
+  <id>com.biglybt.BiglyBT.desktop</id>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
```

Comment:
this is no appropriate for the metadata (this file). `CC0-1.0` is prefered.

Reply:
Used CC0-1.0